truss/contexts/image_builder/serving_image_builder.py

```diff
@@ -114,6 +114,9 @@ def list_bucket_files(bucket_name, data_dir, is_trusted=False):
 
     all_objects = []
     for blob in blobs:
+        # leave out folders
+        if blob.name[-1] == "/":
```

Comment:
Consider using `pathlib.Path` to check whether this is a folder or not, I think it would help the readability here